Makefile.envs

```diff
@@ -59,6 +62,7 @@ export MAIN_MODULE_LDFLAGS= $(LDFLAGS_BASE) -s MAIN_MODULE=1 \
 	$(wildcard $(CPYTHONROOT)/build/bzip2*/libbz2.a) \
 	-lstdc++ \
 	--memory-init-file 0 \
+	-s WASM_BIGINT \
```

Comment:
The only blocker for bigint was Safari 13.X as far as I can tell, and according to https://caniuse.com/bigint the market share of Safari 13.X is now 1.7% as compared to 12%+ for Safari 14.X, so it might be indeed reasonable to start to start requiring it. We might still want explicit feature checks with explicit errors in `pyodide.js` if it's not available. Maybe we can do that in a separate PR though ?

Reply:
Oh yeah I meant to remove that, I added it to see if it would fix the build at some point but it's irrelevant so I removed it but then I reverted the commit that removed it.

Reply:
I'm not really sure about the ABI differences between using `WASM_BIGINT` and not using it, I guess the libffi port was already handling that difference alright before I got to it. But I'm not even sure which way that I was running it because my system for running the libffi tests is a bit ad hoc. I really need to set up CI for that.